### ISSUE TITLE
org.apache.bookkeeper.tests.shaded.BookKeeperServerShadedArtifactTest.testProtobufIsShaded failed

### ISSUE THREAD

**Issue by sijie** [apache/bookkeeper#935]
**BUG REPORT**

1. Please describe the issue you observed:

- What did you do?

run `mvn install` in current master

- What did you expect to see?

it should complete successfully.

- What did you see instead?

The command failed with 

```
Error Message
Expected exception: java.lang.ClassNotFoundException
Stacktrace
java.lang.AssertionError: Expected exception: java.lang.ClassNotFoundException
```

The problem is introduced due to merging #856 after merging #928. Because `pulsar-checksum` depends on guava, which is not included in the changes of #928.

**Comment by athanatos** [comment]
I pushed a PR for this one.

**Comment by sijie** [comment]
@athanatos this is fixed in #937 , because the problem was introduced by `pulsar-checksum` which will be substituted to imported `circe-checksum` in #937. I didn't merge #937 because of current CI job doesn't handle well with paths containing spaces, which I have a fix in #952 .
